model/model_test.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import "testing"
+
+// can run test using : go test -v ./model
+
+func TestGetAllStories(t *testing.T) {
+	stories := GetAllStories()
+
+	if len(stories) != 2 {
+		t.Errorf("Expected number of stories to be 2, but got %d", len(stories))
+	}
```

Comment:
Perhaps we can use the `assert.Equal(t, 2, len(stories), "<The error message you want to show>")`